src/diagnostics/land_compute_methods.jl

```diff
@@ -131,6 +131,20 @@ end
 @diagnostic_compute "soil_net_radiation" SoilCanopyModel p.soil.R_n
 @diagnostic_compute "soil_temperature" SoilCanopyModel p.soil.T
 
+function compute_soil_albedo!(
+    out,
+    Y,
+    p,
+    t,
+    land_model::SoilCanopyModel{FT},
+) where {FT}
+    if isnothing(out)
+        return FT(0.5) .* p.soil.PAR_albedo .+ FT(0.5) .* p.soil.NIR_albedo
+    else
+        @. out = FT(0.5) * p.soil.PAR_albedo + FT(0.5) * p.soil.NIR_albedo
+    end
+end
```

Comment:
Can this be writen as 
```
@diagnostic_compute  "soil_albedo" SoilCanopyModel (p.soil.PAR_albedo .+ p.soil.NIR_albedo) ./ 2
```

Reply:
Yes, this works. The docs for diagnostics, however, say 
"Or, for convenience, you can use the @diagnostic_compute macro which generates the same function. However, it is better to use that macro only if you are getting a defined variable, such as latent heat flux. (without an operation like the bowen ratio above). "

see [docs](https://clima.github.io/ClimaLand.jl/dev/diagnostics/users_diagnostics/)

Reply:
Let's stick with you what you have here. I forgot why I added that comment, but I was problably wiser back then.